src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -0,0 +1,138 @@
+import { LoadingButton } from '@mui/lab';
+import { Box, TextField, Typography } from '@mui/material';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchPerson, updateCristinPerson } from '../../../api/cristinApi';
+import { BackgroundDiv } from '../../../components/styled/Wrappers';
+import { setNotification } from '../../../redux/notificationSlice';
+import { RootState } from '../../../redux/store';
+import { FlatCristinPerson } from '../../../types/user.types';
+import { ResearchProfilePanel } from './ResearchProfilePanel';
+
+type CristinPersonFormData = Pick<FlatCristinPerson, 'background'>;
+
+export const MyFieldAndBackground = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const user = useSelector((store: RootState) => store.user)!;
```

Comment:
Har et persnlig lite problem med å spesifisere at vi vil ignorere regler vi selv har definert :p I tilfellet her håndterer du jo at `user` kan være tom i linja under også, så det trengs ikke.
Forøvrig sånn vi bruker dette andre plasser i dag også
```suggestion
  const user = useSelector((store: RootState) => store.user);
```